specification/hello-world.yaml

```diff
@@ -1,62 +1,94 @@
+     
+     For further details, see the 'Security and authorisation' section below.
+
+    ## API status and roadmap
+    This API is for learning purpose only and is available in the Sandbox environment.
+
+    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/suggestions/107431/hello-world-api-and-tutorials).
+    If you have any other queries, please [contact us](https://digital.nhs.uk/developer/help-and-support).
 
     ## Technology
     This API is [RESTful](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#basic-rest-apis).
-    
+
     ## Network access
     This API is available on the internet and, indirectly, on the [Health and Social Care Network (HSCN)](https://digital.nhs.uk/services/health-and-social-care-network).
     To use this API with NHS smartcards (see below) you do need an HSCN connection, although internet-facing alternatives are available.
-    
+
     For more details see [Network access for APIs](https://digital.nhs.uk/developer/guides-and-documentation/network-access-for-apis).
 
-    ## Authorisation
-    This API uses a variety of different authorisation methods. For details, see the documentation for each endpoint.
+    ## Security and authorisation
+    This API demonstrates a variety of different authentication methods that are used in our APIs.
+
+    ### Open-access authentication 
+    Use this authentication method to access our world endpoint. Some of our APIs are [open-access](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#open-access-apis), which means:
+     - we do not authenticate or authorise the calling application
+     - we do not authenticate or authorise the end user - they might not even be present
+
+    We generally use open-access APIs for public data, where security is not a concern.
+
+    Our open-access APIs are generally RESTful APIs. For more details on how to access them, see our [open-access REST API tutorial](https://digital.nhs.uk/developer/guides-and-documentation/tutorials/open-access-rest-api-tutorial).
 
-    ## Environments and Testing
-    This API is for learning purposes and is only available in our sandbox environment. The base URL for all endpoints is:
-    - `https://sandbox.api.service.nhs.uk/hello/`
+    ### Application-restricted authentication 
+    Use this authentication method to access the application endpoint. This access mode is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
+    You can use this access mode as follows:
+     - unattended (end user not present), for example as part of a back end process to check NHS numbers for data flowing from one system to another
```

Comment:
@SachinPath Hello World doesn't have `unattended` authentication. You can remove this line.

Reply:
Removed unattended authentication.